app/src/main/resources/application.yml

```diff
@@ -3,12 +3,25 @@ server:
   ssl:
     enabled: false
 debug: false
+
 management:
+  security:
+    enabled: false
+  server:
+    port: ${MGMT_PORT:8081}
+    address: ${MGMT_ADDRESS:127.0.0.1}
+  endpoints:
+    enabled-by-default: false
+    web:
+      exposure:
+        include: 'info,health,shutdown'
   endpoint:
     info:
       enabled: true
     health:
       enabled: true
+    shutdown:
+      enabled: ${SHUTDOWN_ENABLED:true}
```

Comment:
I don't think `true` should ever be the default. There are defaults certainly that protect the endpoint with credentials, etc but even then, I see no reason why this should be enabled by default, specially when running this for the public instance.